components/brave_shields/browser/cookie_list_opt_in_service.cc

```diff
@@ -55,4 +67,17 @@ void CookieListOptInService::EnableFilter(bool shouldEnableFilter) {
       brave_shields::kCookieListUuid, shouldEnableFilter);
 }
 
+void CookieListOptInService::OnTooltipShown() {
+  local_state_->SetBoolean(prefs::kAdBlockCookieListOptInShown, true);
+  UMA_HISTOGRAM_EXACT_LINEAR(kCookieListPromptHistogram, 1, 4);
```

Comment:
replace with a function call, see https://chromium.googlesource.com/chromium/src.git/+/HEAD/tools/metrics/histograms/README.md#coding-emitting-to-histograms